src/EncompassRest/Contacts/BorrowerContacts.cs

```diff
@@ -0,0 +1,161 @@
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace EncompassRest.Contacts
+{
+    public sealed class BorrowerContacts
+    {
+        private const string s_apiPath = "encompass/v1/BorrowerContacts";
+
+        #region Public Properties
+        public EncompassRestClient Client { get; }
+        #endregion
+
+        internal BorrowerContacts(EncompassRestClient client)
+        {
+            Client = client;
+        }
+
+        public Task<BorrowerContact> GetContactAsync(string contactId) => GetContactAsync(contactId, CancellationToken.None);
+
+        public Task<BorrowerContact> GetContactAsync(string contactId, CancellationToken cancellationToken) => 
+            GetContactInternalAsync(contactId, cancellationToken, response => response.Content.ReadAsAsync<BorrowerContact>());
+
+        public Task<string> GetContactRawAsync(string contactId) => GetContactRawAsync(contactId, CancellationToken.None);
+
+        public Task<string> GetContactRawAsync(string contactId, CancellationToken cancellationToken) =>
+            GetContactInternalAsync(contactId, cancellationToken, response => response.Content.ReadAsStringAsync());
+
+        private async Task<T> GetContactInternalAsync<T>(string contactId, CancellationToken cancellationToken, Func<HttpResponseMessage, Task<T>> func)
+        {
+            Preconditions.NotNullOrEmpty(contactId, nameof(contactId));
+
+            using (var response = await Client.HttpClient.GetAsync($"{s_apiPath}/{contactId}", cancellationToken).ConfigureAwait(false))
+            {
+                if (!response.IsSuccessStatusCode)
+                {
+                    throw response.StatusCode == HttpStatusCode.NotFound ? await NotFoundException.CreateAsync($"{nameof(GetContactAsync)}/{contactId}", response).ConfigureAwait(false) : await RestException.CreateAsync($"{nameof(GetContactAsync)}", response).ConfigureAwait(false);
+                }
+                return await func(response).ConfigureAwait(false);
+            }
+        }
+
+        public Task<string> CreateContactAsync(BorrowerContact contact) => CreateContactAsync(contact, false, CancellationToken.None);
+
+        public Task<string> CreateContactAsync(BorrowerContact contact, CancellationToken cancellationToken) => CreateContactAsync(contact, false, cancellationToken);
+
+        private Task<string> CreateContactAsync(BorrowerContact contact, bool populate) => CreateContactAsync(contact, populate, CancellationToken.None);
+
+        private Task<string> CreateContactAsync(BorrowerContact contact, bool populate, CancellationToken cancellationToken)
+        {
+            Preconditions.NotNull(contact, nameof(contact));
+
+            return CreateContactInternalAsync(JsonStreamContent.Create(contact), populate ? new QueryParameters(new QueryParameter("view", "entity")).ToString() : null, cancellationToken, async response =>
+            {
+                if (populate)
```

Comment:
Need to set Id and initialize like at https://github.com/EncompassRest/EncompassRest/blob/master/src/EncompassRest/Loans/Loans.cs#L155